Product/Production/Services/DirectCore/src/main/java/gov/hhs/fha/nhinc/direct/DirectAdapter.java

```diff
@@ -140,47 +133,42 @@ private MessageProcessResult processAsDirectMessage(MimeMessage mimeMessage) {
     }
 
     /**
-     * The SmtpAgentConfig.xml is removed as part of CONN-1213. The
-     * direct.appcontext.xml invokes smtpAgent and looks for
-     * ConfigurationService endpoints to be published before the deployment of
-     * Direct and this fails in case of Appservers JBoss,WebLogic and WebSphere
-     * whereas GlassFish works.
+     * The SmtpAgentConfig.xml is removed as part of CONN-1213. The direct.appcontext.xml invokes smtpAgent and looks
+     * for ConfigurationService endpoints to be published before the deployment of Direct and this fails in case of
+     * Appservers JBoss,WebLogic and WebSphere whereas GlassFish works.
      *
-     * CONN-1213: The SmtpAgent is created after the deployment of Direct.
-     * Whenever the DirectSender/DirectReceiver is initiated the SmtpAgent is
-     * created and the aAgent is used for further processing of Direct Messages.
-     * The SmtpAgent injection is removed from direct.beans.xml whereas
-     * ConfigurationService url can be configured in internalConnectionInfo.xml
-     * like any other CONNECT service.
+     * CONN-1213: The SmtpAgent is created after the deployment of Direct. Whenever the DirectSender/DirectReceiver is
+     * initiated the SmtpAgent is created and the aAgent is used for further processing of Direct Messages. The
+     * SmtpAgent injection is removed from direct.beans.xml whereas ConfigurationService url can be configured in
+     * internalConnectionInfo.xml like any other CONNECT service.
      *
-     * Also starts the Agent Settings Manager to refresh agent settings cache
-     * which stores information related to certificates, anchors, domains,
-     * settings etc. The default cache refresh time is 5 minutes and can be
-     * overridden using the gateway property AgentSettingsCacheRefreshTime.
+     * Also starts the Agent Settings Manager to refresh agent settings cache which stores information related to
+     * certificates, anchors, domains, settings etc. The default cache refresh time is 5 minutes and can be overridden
+     * using the gateway property AgentSettingsCacheRefreshTime.
      *
      * @param url the directConfig Service url String converted into URL format.
      * @return SmtpAgent SmtpAgent created to process Direct Messages.
      */
     protected SmtpAgent getSmtpAgent(URL url) {
 
-        if ((smtpAgent != null) && smtpAgent instanceof SmtpAgent) {
+        if (smtpAgent != null) {
```

Comment:
possible to remove null check and use instanceof: 
Please see the link for more information: http://stackoverflow.com/questions/2950319/is-null-check-needed-before-calling-instanceof